dspace-server-webapp/src/main/java/org/dspace/app/rest/ClarinRefBoxController.java

```diff
@@ -331,6 +341,53 @@ private List<ClarinFeaturedServiceLink> mapFeaturedServiceLinks(List<MetadataVal
         return featuredServiceLinkList;
     }
 
+    /**
+     * Remove the unnecessary parts from the output. It is removing the XML header and the <bib:bibtex> tag.
+     */
+    private String updateOutput(String output) {
+        String outputString = "";
+        // Remove <?xml version="1.0" encoding="UTF-8" ?> from the output.
+        if (output != null && output.startsWith(XML_FILE_START)) {
```

Comment:
can we do it more flexible? what if someone adds there a whitspace or similar

Reply:
Updated